playwright/_impl/_transport.py

```diff
@@ -29,7 +29,7 @@ def _get_stderr_fileno() -> Optional[int]:
         # pytest-xdist monkeypatches sys.stderr with an object that is not an actual file.
         # https://docs.python.org/3/library/faulthandler.html#issue-with-file-descriptors
         # This is potentially dangerous, but the best we can do.
-        if not hasattr(sys, "__stderr__"):
+        if not hasattr(sys, "__stderr__") or not sys.__stderr:  # type: ignore
```

Comment:
```suggestion
        if not hasattr(sys, "__stderr__") or not sys.__stderr__: 
```

Reply:
See here, there is the `__stderr__` attribute but it is `None`: https://github.com/microsoft/playwright-python/issues/471#issue-797853555. I try to handle this case with this PR.

Reply:
Your PR is checking `for __stderr` instead of `__stderr__`

Reply:
good catch!